### ISSUE TITLE
`make install` should strip the specs path in myst-gcc

### ISSUE THREAD

**Issue by jxyang** [deislabs/mystikos#155]
After build, we generate a `musl-gcc` executable with the following content:
```bash
#!/bin/sh
exec "${REALGCC:-gcc}" "$@" -specs "<path-to-mystikos-root>/build/musl/lib/musl-gcc.specs"
```

We should strip off ` -specs "<path-to-mystikos-root>/build/musl/lib/musl-gcc.specs`  before copying this file to /opt/mystikos/bin and renaming it to `myst-gcc`.

**Comment by paulcallen** [comment]
Not sure this should be part of installation. It should be done as part of a release build.

**Comment by paulcallen** [comment]
recommend remove from install image. Requires doc change. Can keep for testing, but even for that it may be good to remove.

**Comment by jxyang** [comment]
Fixed in release 0.2.0.
